assets.tf

```diff
@@ -13,12 +13,24 @@ resource "template_dir" "bootstrap-manifests" {
   }
 }
 
-# Self-hosted Kubernetes manifests
-resource "template_dir" "manifests" {
-  source_dir      = "${replace(path.module, path.cwd, ".")}/resources/manifests"
-  destination_dir = "${var.asset_dir}/manifests"
+# copy kubernetes charts yaml to the controller
```

Comment:
This is not copying charts yaml to the controller. That generates `values.yaml` file for the self-hosted Kubernetes chart.